.github/workflows/cli-cache-ee.yml

```diff
@@ -34,6 +34,44 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  inspect-implicit-imports:
+    name: Inspect Implicit Imports
+    runs-on: macos-15
+    steps:
+      - uses: actions/checkout@v4
+        if: ${{ github.event.inputs.commit_sha == '' }}
```

Comment:
instead of just checking for whether a SHA was added ... can we instead check if the user is from Tuist the company (and thus has access to the EE repository)? That way, this pipeline would run automatically for our PRs instead of having to remember to run the EE pipeline manually.